JustSaying.IntegrationTests/JustSayingFluently/GivenANotificationStack.cs

```diff
@@ -84,11 +84,15 @@ protected override IAmJustSayingFluently CreateSystemUnderTest()
                     c.PublishFailureBackoffMilliseconds = _config.PublishFailureBackoffMilliseconds;
                     c.PublishFailureReAttempts = _config.PublishFailureReAttempts;
                 })
-                .WithSnsMessagePublisher<SimpleMessage>()
+                .WithSnsMessagePublisher<SimpleMessage>(cf =>
+                {
+                    cf.BaseTopicName = "this-is-my-topic";
+                })
                 .WithSqsTopicSubscriber()
                 .IntoQueue(TestFixture.UniqueName)
                 .ConfigureSubscriptionWith(cf =>
                 {
+                    cf.BaseTopicName = "this-is-my-topic";

```

Comment:
Should this test set a different name in step 2 to verify that it overwrites the one from step 1?

Reply:
Why have two ways of doing the same thing?

Reply:
I assume the new method is a "quick and simple" way, and the other is the "advanced" way.